### PR TITLE
docs: translate `README.md` to `pt-br`

### PR DESCRIPTION
**Pull request by ImBIOS**
## Summary

Last episode of:

- #1065

## Test Plan

- Please any native or original speaker, review my work. I'm doing it fast, I expect something or some grammar missed.

### CODE REVIEW

File: packages/@biomejs/biome/README.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<p align="center">
+    <img alt="Biome - Toolchain of the web"
+         src="https://raw.githubusercontent.com/biomejs/resources/main/biome-logo-slogan.svg"
+         width="400">
+</p>
+
+<div align="center">
+
+[![Chat no Discord][discord-badge]][discord-url]
+[![CI na principal][ci-badge]][ci-url]
```
Comment (danielsarsi) [inline packages/@biomejs/biome/README.pt-br.md:10]: Maybe should be "CI na `main`"?
```suggestion
[![CI na `main`][ci-badge]][ci-url]
```

File: packages/@biomejs/biome/README.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+[open-vsx-badge]: https://badgen.net/open-vsx/version/biomejs/biome?label=open-vsx&color=green
+[open-vsx-url]: https://open-vsx.org/extension/biomejs/biome
+
+</div>
+
+**Biome** é uma cadeia de ferramentas de alto desempenho para projetos web, visando fornecer ferramentas de desenvolvimento para manter a saúde desses projetos.
+
+**Biome é um [formatador rápido](./benchmark#formatting)** para _JavaScript_, _TypeScript_, _JSX_, e _JSON_ que atinge **[96% de compatibilidade com o _Prettier_](https://console.algora.io/challenges/prettier)**.
+
+**Biome é um [linteiro eficiente](https://github.com/biomejs/biome/tree/main/benchmark#linting)** para _JavaScript_, _TypeScript_, e _JSX_ que possui **[mais de 170 regras](https://biomejs.dev/linter/rules/)** do ESLint, TypeScript ESLint, e [outras fontes](https://github.com/biomejs/biome/discussions/3).
```
Comment (danielsarsi) [inline packages/@biomejs/biome/README.pt-br.md:32]: I suggest using "linter", as "linteiro" is an foreignism.
```suggestion
**Biome é um [linter eficiente](https://github.com/biomejs/biome/tree/main/benchmark#linting)** para _JavaScript_, _TypeScript_, e _JSX_ que possui **[mais de 170 regras](https://biomejs.dev/linter/rules/)** do ESLint, TypeScript ESLint, e de [outras fontes](https://github.com/biomejs/biome/discussions/3).
```

File: packages/@biomejs/biome/README.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+# verificar todos os arquivos contra formatação, lint, etc. em ambientes CI
+npx @biomejs/biome ci ./src
+```
+
+Se você quiser experimentar o Biome sem instalá-lo, use o [playground online](https://biomejs.dev/playground/), compilado para WebAssembly.
+
+## Documentação
+
+Confira nossa [página inicial][biomejs] para saber mais sobre o Biome,
+ou vá diretamente para o [Guia de Introdução][getting-started] para começar a usar o Biome.
```
Comment (danielsarsi) [inline packages/@biomejs/biome/README.pt-br.md:65]:
```suggestion
ou vá ao [Guia de Introdução][getting-started] para começar a usar o Biome.
```

File: packages/@biomejs/biome/README.pt-br.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+[npm-badge]: https://badgen.net/npm/v/@biomejs/biome?icon=npm&color=green&label=%40biomejs%2Fbiome
+[npm-url]: https://www.npmjs.com/package/@biomejs/biome/v/latest
+[vscode-badge]: https://badgen.net/vs-marketplace/v/biomejs.biome?label=vscode&icon=visualstudio&color=green
+[vscode-url]: https://marketplace.visualstudio.com/items?itemName=biomejs.biome
+[open-vsx-badge]: https://badgen.net/open-vsx/version/biomejs/biome?label=open-vsx&color=green
+[open-vsx-url]: https://open-vsx.org/extension/biomejs/biome
+
+</div>
+
+**Biome** é uma cadeia de ferramentas de alto desempenho para projetos web, visando fornecer ferramentas de desenvolvimento para manter a saúde desses projetos.
```
Comment (danielsarsi) [inline packages/@biomejs/biome/README.pt-br.md:28]:
```suggestion
**Biome** é um conjunto de ferramentas de alto desempenho para projetos web, visando fornecer recursos de desenvolvimento para manter a saúde desses projetos.
```

**Review by danielsarsi — COMMENTED** [review]
Leaving my suggestions here.

File: packages/@biomejs/biome/README.pt-br.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+**Biome é um [formatador rápido](./benchmark#formatting)** para _JavaScript_, _TypeScript_, _JSX_, e _JSON_ que atinge **[96% de compatibilidade com o _Prettier_](https://console.algora.io/challenges/prettier)**.
+
+**Biome é um [linteiro eficiente](https://github.com/biomejs/biome/tree/main/benchmark#linting)** para _JavaScript_, _TypeScript_, e _JSX_ que possui **[mais de 170 regras](https://biomejs.dev/linter/rules/)** do ESLint, TypeScript ESLint, e [outras fontes](https://github.com/biomejs/biome/discussions/3).
+Ele **fornece diagnósticos detalhados e contextualizados** que ajudam você a melhorar seu código e se tornar um programador melhor!
+
+**Biome** é projetado desde o início para ser usado [interativamente dentro de um editor](https://biomejs.dev/guides/integrate-in-editor/).
+Pode formatar e lintar código malformado enquanto você está escrevendo.
```
Comment (CookieDasora) [inline packages/@biomejs/biome/README.pt-br.md:36]: "Isso permite formatar e lintar códigos malformados enquanto você programa."

**Review by CookieDasora — COMMENTED** [review]
Thanks @ImBIOS for your contribution, I left some suggestions.

**Comment by ImBIOS** [comment]
Thanks for the feedback. Very helpful for me.

File: packages/@biomejs/biome/README.ja.md [hunk 6]
```diff
@@ -112,9 +112,8 @@ Biome についてもっと知るために[ホームページ][biomejs]をチェ
 - [English](./README.md)
 - [简体中文](./README.zh-CN.md)
 - 日本語
+- [Português do Brasil](./README.pt-br.md)
 
-[bench]: https://github.com/biomejs/biome/blob/main/benchmark/README.md
 [biomejs]: https://biomejs.dev/ja/
 [biome-philosophy]: https://biomejs.dev/ja/internals/philosophy/
 [language-support]: https://biomejs.dev/ja/internals/language-support/
-[getting-started]: https://biomejs.dev/ja/guides/getting-started/
```
Comment (unvalley) [inline packages/@biomejs/biome/README.ja.md:120]: This line is needed to link, but bench can be deleted because it's unused.

**Review by unvalley — CHANGES_REQUESTED** [review]
Thank you for translation!
I can't review Brazilian Portuguese language, thank you for reviewers! 
We can merge this after fixing my request.

File: packages/@biomejs/biome/README.pt-br.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
+<p align="center">
+    <img alt="Biome - Cadeia de ferramentas da web"
```
Comment (victor-teles) [inline packages/@biomejs/biome/README.pt-br.md:2]:
```suggestion
    <img alt="Biome - Conjunto de ferramentas da web"
```
Previously we translated this into "Conjunto de ferramentas", so would be good to keep the same here

**Review by victor-teles — CHANGES_REQUESTED** [review]

**Review by danielsarsi — APPROVED** [review]
Seems good to me!

**Review by unvalley — APPROVED** [review]

**Review by victor-teles — APPROVED** [review]

**Pull request merged**